### PR TITLE
[PINS] Endpoint createPin

### PR DESCRIPTION
**Pull request by HectorxSM317**
AS: Admin
I WANT TO create one pin in to panel admin
SO I can see information the pin created

Acceptance Criteria:

When the admin needs create one pin, he will be able to do it and see al information success.

EVIDENCE:

success!

image

in case of error

image

### CODE REVIEW

**Comment by LucasVaroneW** [comment]
La rama tiene conflictos, deberían solucionarse

File: controllers/Pines/createPin.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      const { swap, typeCode, status, code, typeCard } = req.body;
+
+      const newPin = await Pin.create({
+        swap,
+        typeCode,
+        status,
+        code,
+        typeCard,
+      });
+      console.log("asdsad");
```
Comment (Jair-Academlo) [inline controllers/Pines/createPin.js]: delete console.log

**Review by Jair-Academlo — APPROVED** [review]

**Comment by Jair-Academlo** [comment]
delete console.log resolved conflicts

**Comment by NevercomeX** [comment]
Conflicts solved

**Pull request merged**